src/lib/sample_metadata.rs

```diff
@@ -788,4 +811,51 @@ Sample2,GGGG
         ];
         assert_eq!(actual, expected);
     }
+    #[test]
+    fn test_get_semantic_barcode() {
```

Comment:
@nh13 Moved and renamed this test after your suggestion to move the method into `SampleMetadata`